### DISCUSSION TITLE
set oMEXPX for night lapse photo

### DISCUSSION THREAD

**Discussion by tomas123** [gopro/labs#497] [Q&A] [answered]
> Enhanced EXTQ,EXPN,EXPX and EXPT to work on timelapse video and photo

How can I set a time over 1 second in EXPX for night lapse photo (Hero 11)?
I tried without success (for 2 seconds):
`oMEXPX=0.5`

There is a syntax for time-lapse frame rates:
```
p2 (0.5s or 2Hz/2p capture)
p.2 (2s interval)
```
but this also doesn't work
`oMEXPX=.2`

**Comment by dnewman-gpsw** [comment, marked as answer]
EXPX was never thought to be used in non-video modes, so when extended to photo, the inputs are still integers. So 1 is the smallest input this control allows, limiting the max night photo exposure to 1 second.  I will see if this can be upgraded to use float inputs.
